examples/gallery/maps/country_polygons.py

```diff
@@ -0,0 +1,100 @@
+"""
+Highlight country and continent polygons
+----------------------------------------
+The :meth:`pygmt.Figure.coast` method can
+highlight country polygons via the ``dcw``
+parameter. It accepts the country code or
+full country name and can draw its borders
+and add a color to its landmass. It's also
+possible to define multiple countries at
+once by separating the indiviudal names
+with commas.
```

Comment:
Currently each line is only about ~40 characters long, but 80 characters are allowed. Please reformat these lines to make the file shorter.

Reply:
Done in [c965eb8](https://github.com/GenericMappingTools/pygmt/pull/2428/commits/c965eb84c454a273759c8fcabd587d398e117953).